src/interfaces.ts

```diff
@@ -134,5 +135,5 @@ export interface MetricData<T extends MetricSignature = MetricSignature>
 
 export type Conditions = [
   (event: SignedTriggerEvent) => boolean,
-  (event: SignedTriggerEvent) => Promise<MetricData>,
+  (event: SignedTriggerEvent, repoClient: Octokit) => Promise<MetricData>,
```

Comment:
to avoid the unused var in some metrics make the new parameter optional:
```suggestion
  (event: SignedTriggerEvent, repoClient?: Octokit) => Promise<MetricData>,
```

Reply:
Accepted.